webpack.config.js

```diff
@@ -35,16 +35,15 @@ if ( isProduction ) {
 			css: [
 				'./node_modules/tailwindcss/dist/base.css'
 			],
-			extractors: [
-				{
-					extractor: class TailwindExtractor {
-						static extract(content) {
-							return content.match(/[A-Za-z0-9-_:\/]+/g) || [];
-						}
-					},
-					extensions: ['php', 'js', 'svg', 'css',]
-				}
-			],
+			defaultExtractor: content => {
+    				// Capture as liberally as possible, including things like `h-(screen-1.5)`
+    				const broadMatches = content.match(/[^<>"'`\s]*[^<>"'`\s:]/g) || []
+
+    				// Capture classes within other delimiters like .block(class="w-1/2") in Pug
```

Comment:
Is this newer syntax for Tailwind? I don't remember class names like this. Please include a link to some docs. Thanks! This also includes the above option.

Reply:
purgecss changed the way they define extractors.That's why I decided to copy the ones that tailwind recommends in their docs: https://tailwindcss.com/docs/controlling-file-size#setting-up-purge-css-manually